src/utils.js

```diff
@@ -0,0 +1,14 @@
+export const addSeparators = (strings, and) =>
+  strings.reduce((accumulator, string, i) => {
+    if (i === 0) {
+      return `${accumulator}${string}`;
+    }
+
+    if (i === strings.length - 1) {
+      return `${accumulator} ${and} ${string}`;
```

Comment:
```suggestion
      return `${accumulator} ${and} ${string}`;
```
(replace ordinary space with no-break space character ` `)

Reply:
Fixed.